### PR TITLE
Dyn 2560 models folder2

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Increase the code coverage adding test cases for the DynamoCore/Models folder second part.
I created several unit tests using nunit and AxoCover (code coverage) for the next classes:

CreateAndConnectNodeCommand
CreateCustomNodeCommand
CreateNodeCommand
CreateNoteCommand
CreateProxyNodeCommand
ForceRunCancelCommand
GroupModelCommand
ModelEventCommand
MutateTestCommand
OpenFileCommand
PausePlaybackCommand
PresetCommand
RecordableCommands
SelectModelCommand
UndoRedoCommand
UpdateModelValueCommand

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@QilongTang 

### FYIs

@alfredo-pozo

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
This pull request contains just test methods for the classes inside the RecordableCommands.cs file
I got more than 94% of the coverage in the RecordableCommands.cs file.

image

**Comment by RobertGlobant20** [comment]
This is a screenshot after running all the test cases in the DynamoCoreTest project in Visual Studio, I got the usual 67 cases failed and 2213 passed. Also I ran the DynamoSelfServe job with my branch in master-5 and it finished successfully.

image

File: test/DynamoCoreTests/Models/PresetCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        [Category("UnitTests")]
+        public void AddPresetCommand_DeserializeCore()
+        {
+            //Arrange
+            var ids = DynamoSelection.Instance.Selection.OfType<NodeModel>().Select(x => x.GUID).ToList();
+            var AddPresetCommand = new DynamoModel.AddPresetCommand("PresetName", "Description", ids);
+            XmlDocument xmlDocument = new XmlDocument();        
+            XmlElement elemTest = xmlDocument.CreateElement("TestCommand");
+
+            //Act//Assert
```
Comment (QilongTang) [inline test/DynamoCoreTests/Models/PresetCommandTests.cs:32]: Copy paste error here?

Comment (RobertGlobant20) [reply test/DynamoCoreTests/Models/PresetCommandTests.cs]: @QilongTang I've fixed this comment and in other 2 test methods.

Please let me know if you have any other comment.

commit:
https://github.com/DynamoDS/Dynamo/pull/10640/commits/d5f1a2d07e79a4e251949d8f85b953bb8d8cbda6

**Pull request merged**